aad/Module.md

```diff
@@ -1,91 +1,33 @@
 ## Overview
-Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language 
-easily. It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent 
+Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language. 
```

Comment:
Let;s use Active Directory (AD) in place of the first AD.

Reply:
Added